glotaran/project/scheme.py

```diff
@@ -75,6 +76,15 @@ def markdown(self):
 
         return MarkdownStr(markdown_str)
 
+    def grouped(self) -> bool:
```

Comment:
Nitpicky.
Renaming it to `is_goupable` would make more sense.
Could be done in a different PR.